### ISSUE TITLE
Hey So I have a few questions

### ISSUE THREAD

**Issue by rayanmargham** [4mbr0s3-2/FNF-Groovin-Docs#1]
Hey @4mbr0s3-2 !

Recently I discovered a lot of your work and to say I'm blown away is an understatement, there are so many things that I cannot BEGIN to talk about. How the hell was that even possible in HaxeFlixel?!?!? but I'm getting off topic.

You see I'm working on a rewrite of the entire game from scratch using Rust and a Custom Game Engine. there were a few things that have me scratching my head on how exactly would I implement the necessary backend functions. For Example: You had the strumline and notes rotate in a way that creates a trippy 3D effect and while I plan to implement 3D Notes and StrumLines since my Engine uses 3D I'm still confused on how it was achieved.

in Summary is it possible for you to explain the underlying behind the scenes behind some of your FNF projects?

Please Get in Contact with me
